stories/Playground/Playground.story.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import {description, header} from 'wix-storybook-utils/Sections';
+
+import Playground from 'wix-storybook-utils/Playground';
+
+import { allComponents } from '../utils/allComponents';
+// @ts-ignore
+import exampleCode from '!raw-loader!./example';
+import { Tabs, ALIGNMENT } from '../../src/components/Tabs';
+
+const LAYOUT_INDEX_MAP = {
+    0: 'vertical',
+    1: 'horizontal'
+}
+
+const PlaygroundWrapper = () => {
+    const [layoutIndex, setLayoutIndex] = React.useState(0);
+    const isHorizontalLayout = layoutIndex === 1;
+
+    return (
+        <div
+            style={{
+                /* expand playground area to be wider than parent element */
+                /* keep 5% distance from screen edge on left and right */
+                margin: `0 calc((100vw / 2 - 100% / 2) * -1 + 5%)`,
+            }}
+        >
+            <Playground
+                key={LAYOUT_INDEX_MAP[layoutIndex]}
+                compact={isHorizontalLayout}
+                initiallyOpen={isHorizontalLayout}
+                initialCode={exampleCode}
+                scope={allComponents}
+                formatSnippetUrl={id => `${window.parent.location.href}&snippet=${id}`}
+            />
+            <Tabs
+                alignment={ALIGNMENT.right}
+                activeTabIndex={layoutIndex}
+                items={[
+                    { title: 'Vertical layout' },
+                    { title: 'Horizontal layout' }
+                ]}
+                onTabClick={(index) => setLayoutIndex(index)}
+            />
```

Comment:
Should this maybe be on the top?
And maybe align left?
I'm asking out loud.
Maybe @johannaasseraf or @michalvardi have an input on this

Reply:
I was also debating about the alignment and where it should be placed. Finally, I kept it the same as in WSR but we could change it..